js/checklist.js

```diff
@@ -148,10 +163,10 @@
 			url: BASE_COURSE_URL + "/external_tools/" + POLICY_WIZARD_TOOL_ID,
 			iconClass: 'icon-educators'
 		});
-		
+
 	
 		//----- DEBUG -----
-		if(DEBUG) {
+		if(false) {
```

Comment:
This is being left in there so the dev can tweak it and doesn't need to rewrite the logic each time?